<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ReservationsUI</class>
    <widget class="QWidget" name="ReservationsUI">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
                <widget class="QGroupBox" name="groupBox">
                    <property name="title">
                        <string>Reservation Details</string>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                        <item row="0" column="0">
                            <widget class="QLabel" name="nameLabel_2">
                                <property name="text">
                                    <string>ID:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLineEdit" name="idLineEdit"/>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="clientLabel">
                                <property name="text">
                                    <string>Client:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QComboBox" name="clientComboBox"/>
                        </item>
                        <item row="2" column="0">
                            <widget class="QLabel" name="seatLabel">
                                <property name="text">
                                    <string>Seat:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="1">
                            <widget class="QComboBox" name="seatComboBox"/>
                        </item>
                        <item row="3" column="0">
                            <widget class="QLabel" name="dateTimeLabel">
                                <property name="text">
                                    <string>Date/Time:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="1">
                            <widget class="QDateTimeEdit" name="dateTimeEdit"/>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QTableView" name="reservationsTableView"/>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QPushButton" name="addReservationButton">
                            <property name="text">
                                <string>Add Reservation</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="deleteReservationButton">
                            <property name="text">
                                <string>Delete Reservation</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="updateReservationButton">
                            <property name="text">
                                <string>Update Reservation</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>